### ISSUE TITLE
MAS 4.1.3 - Status Messages

### ISSUE THREAD

**Issue by scottbommarito** [NuGet/NuGetGallery#7533]
Status messages should be properly announced by the screen reader without shifting focus.
In other words, if content that is not focused changes, or the result of an action is displayed on the screen but is not focused, screen readers should read this content as an alert.

An good example of this on our site right now is that when you search for packages, the site announces "There are X packages that fit your criteria" without focus.

**Comment by scottbommarito** [comment]
I don't see any violations of this on the site--we use `role="alert"` whenever appropriate.
